dev_requirements.txt

```diff
@@ -0,0 +1,7 @@
+flake8==3.9.2
```

Comment:
These are all requirements tools to make developing of the bot easier.
See development_setup.md